src/dictionaries/parseAndBuild-1000.js

```diff
@@ -0,0 +1,101 @@
+const request = require("request");
```

Comment:
There's no need to include this script since we aren't going to be running it in the main repo